### DISCUSSION TITLE
I have uploaded a new panel and...

### DISCUSSION THREAD

**Discussion by drnotube** [RomanKubiak/ctrlr#133] [General]
I have uploaded a new panel and I don't see it on the web (https://ctrlr.org/panels/)
Have I missed a step to complete?
the panel is here:
https://ctrlr.org/yamahadjxii-vst-panel/

**Comment by dobo365** [comment]
Sorry, what is your issue? When going to the page one can see the panel...

**Reply by drnotube** [reply]
yes, but I don't see it in the main panel list: https://ctrlr.org/panels/

**Comment by RomanKubiak** [comment]
i'll be reworking this, the plugin the generated that /panels/ page is not working properly i'll try to fix it soon
